src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import * as S from './SubscriberGdpr.styles';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { InjectAppServices } from '../../../services/pure-di';
+
+const PermissionValue = ({ value }) => {
+  const permissionValue = value.toLowerCase();
+  const iconColor =
+    permissionValue === 'none' ? 'grey' : permissionValue === 'true' ? 'green' : 'red';
+  const valueText =
+    permissionValue === 'none'
+      ? 'Sin respuesta'
+      : permissionValue === 'true'
+      ? 'Aceptado'
+      : 'Rechazado';
+
+  return (
+    <div className="dp-icon-wrapper">
+      <span className={`ms-icon icon-lock dp-lock-${iconColor}`} />
+      {valueText}
+    </div>
+  );
+};
+
+const PermissionExpandableRow = ({
+  field,
+  dependencies: {
+    appConfiguration: { reportsUrl },
+    experimentalFeatures,
+  },
+}) => {
+  const [expanded, setExpanded] = useState(false);
+  const isPermissionHistoryEnabled =
+    experimentalFeatures && experimentalFeatures.getFeature('PermissionHistory');
+
+  /**
+   * Toggle the expanded state of the row and fetch the permissions if the row is expanded.
+   */
+  const toggleExpanded = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <>
+      <tr>
+        <td>
+          <span className="dp-name-text">
+            {isPermissionHistoryEnabled && field.value != 'none' && (
+              <button
+                type="button"
+                onClick={toggleExpanded}
+                className={`dp-expand-results ${expanded && 'dp-open-results'}`}
+              >
+                <i className="ms-icon icon-arrow-next"></i>
+              </button>
+            )}
+            {field.name}
+          </span>
+        </td>
+        <td>
+          {field.permissionHTML ? (
+            <S.TextColumn dangerouslySetInnerHTML={{ __html: field.permissionHTML }} />
```

Comment:
If I am not wrong, this should have quotes.

I am not sure about how to implement it, it is only an illustration:

```suggestion
            <S.TextColumn dangerouslySetInnerHTML={{ __html: `"${field.permissionHTML}"` }} />
```

Reply:
I haven't seen that in the UI library 🤔 
![image](https://user-images.githubusercontent.com/25908890/112325835-669ef100-8c8a-11eb-983e-627cb4d5cec3.png)

Reply:
En realidad la librería de UI es solo eso, estilos para que puedan usarse en la aplicación, tiene una demo, pero no es un mockup para reproducir en la app.

Creo que las comillas tienen sentido porque es texto literal escrito por el usuario, las veo en la [solicitud de presupuesto](https://docs.google.com/document/d/1glGJbm4YxZDs4W42v7vQexmIV_GKEQKrocUStn18Jf0/edit) y en [Confluence](https://makingsense.atlassian.net/wiki/spaces/DOP/pages/761561444/Historial+de+Permisos+RGPD+del+Suscriptor). Pero lo mejor sería que lo confirmes con Guille, tal vez en [el chat](https://makingsense.slack.com/archives/C01PLPL2UAU).

Reply:
Por otro lado, en la implementación anterior, las comillas no estaban:

![image](https://user-images.githubusercontent.com/1157864/112525314-88c66b00-8d7f-11eb-90b5-8b10270cbcd0.png)

Por lo tanto, podría considerarse fuera del scope de esta tarea, si alguién quiere las comillas, que las pida 😛.

Creo que podríamos dar este tema por resuelto.